app/models/dictionary.rb

```diff
@@ -34,6 +34,22 @@ def self.gettext(text, opts = {})
     opts[:plural] ? col.send(opts[:notfound]).send(:pluralize) : col.send(opts[:notfound])
   end
 
+  def self.ui_lookup(option = {})
```

Comment:
Looks like a typo, should be `options = {}`

Reply:
Good catch.  Thanks!